assets/js/components/DashboardMainApp.js

```diff
@@ -127,10 +127,24 @@ export default function DashboardMainApp() {
 			const widgetClass = `.googlesitekit-widget-area--${ widgetArea }`;
 
 			setTimeout( () => {
-				global.scrollTo( {
-					top: getNavigationalScrollTop( widgetClass, breakpoint ),
-					behavior: 'smooth',
-				} );
+				function scrollToWidgetArea() {
+					global.scrollTo( {
+						top: getNavigationalScrollTop(
+							widgetClass,
+							breakpoint
+						),
+						behavior: 'smooth',
+					} );
+				}
+
+				function handleScrollEnd() {
+					scrollToWidgetArea();
+					global.removeEventListener( 'scroll', handleScrollEnd );
```

Comment:
Shouldn't this be `scrollend`?

```suggestion
					global.removeEventListener( 'scrollend', handleScrollEnd );
```

Reply:
Oops!
That was my mistake @nfmohit . Thanks for spotting it.

I have changed this to remove the `scrollend` event as per the suggestion.